### DISCUSSION TITLE
Syntax highlighting for response body should be configurable with ScalarOptions

### DISCUSSION THREAD

**Discussion by redstrike** [scalar/scalar#3514] [Ideas]
I like reading colored JSON response bodies. However, syntax highlighting truly affects the scrolling and overall rendering performance of the API Reference. It has been improved by lower the threshold to 20k characters recently, but I still able to notice a high latency rendering while switching between endpoints for testing or displaying a big JSON response.

It would be great if we can configure the syntax highlighting via ScalarOptions with a lower max characters threshold for enabling the highlighting or just disable it completely because it affects the scrolling performance significantly.

**Comment by amritk** [comment]
Yep that is totally possible. So currently its set to 20K bytes actually, we switched to to size recently. 

@hanspagel @marclave do we want to option this in? Or just lower the threshold?
